src/models/slices/query.ts

```diff
@@ -1,10 +1,32 @@
 import { QuerySource, QueryTrigger, SearchIntent } from '@yext/answers-core';
 
+/**
+ * Maintains the latest query and its associated data.
+ */
 export interface QueryState {
+  /**
+   * The user input used for the next search query.
+   */
   input?: string,
+  /**
+   * The ID of the query from the latest search.
+   */
   queryId?: string,
+  /**
+   * How the query was triggered (besides user input).
```

Comment:
would we want to use `@inheritDoc` for any of these? Not sure if it works, but maybe it could simplify some things?

Reply:
yes, I've been using `@inheritDoc` in some places. I didn't use it for `queryTrigger` because the current description felt a little more direct/simple to me. but, if we want the extra detail, I can use `{@inheritDoc QueryTrigger}` instead